app/src/main/res/layout/fragment_edit_profile.xml

```diff
@@ -102,21 +102,43 @@
             android:inputType="text" />
     </com.google.android.material.textfield.TextInputLayout>
 
-    <com.google.android.material.textfield.TextInputLayout
-        style="@style/Widget.MaterialComponents.TextInputLayout.OutlinedBox.Dense"
-        android:layout_width="match_parent"
-        android:layout_height="wrap_content"
-        android:paddingBottom="@dimen/padding_medium"
-        android:hint="@string/phone">
-
-        <com.google.android.material.textfield.TextInputEditText
-            android:id="@+id/phone"
+        <LinearLayout
             android:layout_width="match_parent"
             android:layout_height="wrap_content"
-            android:inputType="number" />
-    </com.google.android.material.textfield.TextInputLayout>
+            android:orientation="horizontal"
+            android:paddingBottom="@dimen/padding_medium"
+            android:weightSum="1">
 
-    <com.google.android.material.textfield.TextInputLayout
+            <com.hbb20.CountryCodePicker
+                android:id="@+id/countryCode"
+                android:layout_width="0dp"
+                android:layout_height="wrap_content"
+                android:layout_weight=".2"
+                app:ccp_autoDetectCountry="true"
+                app:ccp_rememberLastSelection="true"
+                app:ccp_showFlag="false"
+                app:ccp_showNameCode="false" />
+
+
+            <com.google.android.material.textfield.TextInputLayout
+                style="@style/Widget.MaterialComponents.TextInputLayout.OutlinedBox.Dense"
+                android:layout_width="0dp"
+                android:layout_height="wrap_content"
+                android:layout_weight=".8"
+                android:hint="@string/phone">
+
+                <com.google.android.material.textfield.TextInputEditText
+                    android:id="@+id/phone"
+                    android:layout_width="match_parent"
+                    android:layout_height="wrap_content"
+                    android:inputType="phone"
+                    android:maxLength="10" />
```

Comment:
Mobile numbers can be of variable length in different countries. I think it shouldn't be apt to limit it to 10 as you have added spinner of mobile code for different countries.

Reply:
@iamareebjamal should I then increase the limit to 15 digits?

Reply:
??

> @iamareebjamal should I then increase the limit to 15 digits?

Reply:
I don't know. Extremely low priority issue. I unfortunately have a lot on my plate and have to deal with much much more important bugs and PRs

Reply:
Ok